server/models/Collective.js

```diff
@@ -2135,17 +2135,31 @@ export default function (Sequelize, DataTypes) {
     });
   };
 
-  Collective.prototype.getBalance = function (until) {
+  Collective.prototype.getBalance = async function (until) {
     until = until || new Date();
-    return models.Transaction.findOne({
+    const transactions = await models.Transaction.findOne({
       attributes: [
         [Sequelize.fn('COALESCE', Sequelize.fn('SUM', Sequelize.col('netAmountInCollectiveCurrency')), 0), 'total'],
       ],
       where: {
         CollectiveId: this.id,
         createdAt: { [Op.lt]: until },
       },
-    }).then(result => Promise.resolve(parseInt(result.toJSON().total, 10)));
+    });
+    const transactionSum = parseInt(transactions.toJSON().total, 10);
+
+    // Subtract expenses scheduled for payment
+    const expenses = await models.Expense.findOne({
+      attributes: [[Sequelize.fn('COALESCE', Sequelize.fn('SUM', Sequelize.col('amount')), 0), 'total']],
+      where: {
+        CollectiveId: this.id,
+        status: expenseStatus.SCHEDULED_FOR_PAYMENT,
```

Comment:
Should we also include `PROCESSING` in there?

If the collective has a $100 balance, we should not allow to have two $75 expenses processing at the same time.

Reply:
Yes, that's coming in the workers PR, already implemented.
In the mid-term, I also want to migrate TransferWise to this implementation, I'm currently creating transactions before finishing processing.